services/121-service/test/helpers/registration.helper.ts

```diff
@@ -267,37 +268,6 @@ export function getRegistrations({
     .send();
 }
 
-export async function changeBulkRegistrationStatus({
```

Comment:
Was this not used anymore anywhere? 🎉 :)

Reply:
yup